HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.2 / 2021-07-20
+
+- Edit gitignore
```

Comment:
Maybe `Cleaned up gitignore`? I think that's what you said in most of these PRs.

Reply:
I was using 'cleaned up' to refer to the ones that used to be the entire template, and then 'edit' or 'update' for the ones that were already language specific but needed to be tweaked slightly. so among all the changes, this one (and a couple others) are worded differently, but within the repo itself, this wording makes a bit more sense, I think, than 'cleaned up'.